openy.install

```diff
@@ -52,3 +52,15 @@ function openy_update_8002() {
   // Enable module.
   \Drupal::service('module_installer')->install(['openy_node_alert']);
 }
+
+/**
+ * Enable config_import module.
+ */
+function openy_update_8003() {
+  if (\Drupal::service('module_handler')->moduleExists('config_import')) {
+    return;
```

Comment:
Early return pattern makes sense only if amount of code more than 1 line )))
```php
if (!\Drupal::service('module_handler')->moduleExists('config_import')) {
  // Enable module.
  \Drupal::service('module_installer')->install(['config_import']);
}
```
But in this case I believe no need to check for module installed at all
Drupal has that check, so just enable it and let the core makes its job